packages/palette/src/elements/Typography/Typography.tsx

```diff
@@ -184,6 +185,10 @@ function createStyledText<P extends StyledTextProps>(
             fontFamilyType && themeProps.fontFamily[fontType][fontFamilyType]
           }
           {...determineFontSizes(fontType, size)}
+          // styled-components wants it called `as`, but this function somehow
+          //  gets skipped when there is an `as` prop passed in. So our clients
+          //  will pass in `element`, and we'll pass it through as `as`.
+          {...(element ? { as: element } : {})}
```

Comment:
Do you tink we should stick to styled-components conventions? Is it a bug that `as` isn't getting picked up?